ai_worker/tests/src/test_image_vision.py

```
@@ -0,0 +1,335 @@
+        self.assertGreater(len(analysis.atmosphere), 0)
+
+
+class TestCombinedOCRAndVision(unittest.TestCase):
+    """OCR + Vision 통합 분석 테스트"""
+
+    @patch('src.parsers.image_vision.openai')
+    @patch('src.parsers.image_vision.pytesseract')
+    @patch('src.parsers.image_vision.Image')
+    @patch('src.parsers.image_vision.Path')
+    @patch('builtins.open', new_callable=mock_open, read_data=b'image data')
+    def test_parse_with_ocr_and_vision(self, mock_file, mock_path, mock_image, mock_tesseract, mock_openai):
```

Pylint (4.0.3)

[refactor] Too many positional arguments (6/5)

(R0917)